tools/setup/emoji/build_emoji

```diff
@@ -204,6 +206,8 @@ def dump_emojis(cache_path):
     code_point_to_fname_map = code_point_to_file_name_map(EMOJI_DUMP_PATH("NotoColorEmoji.ttx"))
     emoji_data = ujson.load(open(os.path.join(EMOJI_DATA_PATH, 'emoji.json')))
     emoji_catalog = generate_emoji_catalog(emoji_data)
+    UNIFIED_REACTIONS_PATH = os.path.join(ZULIP_PATH, 'zerver', 'management', 'data', 'unified_reactions.json')
+    unified_reactions_data = ujson.load(open(UNIFIED_REACTIONS_PATH))
 
```

Comment:
This is probably not a big deal, but shouldn't the file object created by `open(UNIFIED_REACTIONS_PATH)` be closed? Maybe use the `with statement`?